src/custom/pages/Claim/styled.ts

```diff
@@ -1179,3 +1179,37 @@ export const CowSpinner = styled.div`
     }
   }
 `
+
+export const StepExplainer = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 0 16px;
+  margin: 0 0 24px;
+
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    grid-template-columns: 1fr;
+    gap: 16px 0;
+  `}
+  > span {
+    padding: 24px;
+    background: ${({ theme }) => theme.blueShade3};
+    border-radius: 12px;
+    position: relative;
+  }
+
+  > span::before {
+    content: 'Step ' attr(data-step);
```

Comment:
i don't personally like this it makes it quite hard to change later on if we forget it's in the CSS. Can't you just write this out in the HTML of `InvestmentFlow`?

Reply:
@W3stside Agreed, addressed now.